extra/contrib/fusion_rgb_cycle.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
```

Comment:
Please add a suitable copyright notice and SPDX license header in a comment or module docstring.

It should look something like:

```
Copyright (C) 2022–2022  Peter Eckersley
SPDX-License-Identifier: GPL-3.0-or-later
```